node_pools.tf

```diff
@@ -100,12 +100,6 @@ resource "google_container_node_pool" "node_pool_mt" {
 
   version = var.kube_version_gke
 
-  # We want the multi-tenant node pool to be completely replaced
-  # instead of rolling deployment.
-  # The master_version will ensure that the node pool is created then
-  # destroyed if there is an update.
-  name = "${var.deployment_id}-mt-${formatdate("MM-DD-hh-mm", timestamp())}"
```

Comment:
I have found not setting the name is better than setting it. we can delete this now because blue node pool is off in prod cloud